rag_experiment_accelerator/doc_loader/documentIntelligenceLoader.py

```diff
@@ -88,13 +87,13 @@ def load_with_azure_document_intelligence(
     logger.debug(f"Loaded {len(documents)} documents using Azure Document Intelligence")
 
     text_splitter = RecursiveCharacterTextSplitter(
-        chunk_size=chunk_size,
-        chunk_overlap=overlap_size,
+        chunk_size=index_config.chunking.chunk_size,
+        chunk_overlap=index_config.chunking.overlap_size,
```

Comment:
Severity: Major - Accessing `index_config.chunking.chunk_size` and `index_config.chunking.overlap_size` without validation can lead to runtime errors if these attributes are missing or invalid. Implement checks to ensure these attributes exist and are within acceptable ranges.